src/status_im/contexts/wallet/common/utils/send.cljs

```diff
@@ -20,3 +20,13 @@
 (defn calculate-full-route-gas-fee
   [route]
   (reduce money/add (map calculate-gas-fee route)))
+
+(defn find-affordable-networks
+  [{:keys [balances-per-chain input-value selected-networks disabled-chain-ids]}]
+  (->> balances-per-chain
+       (filter (fn [[_ {:keys [balance chain-id]}]]
+                 (and
+                  (>= (js/parseFloat balance) input-value)
```

Comment:
Not sure, I didn't write this myself but moved the code from other part of the codebase, but I'll check if we can use big number as it would be safer 👍